verifier-guest.sh

```diff
@@ -19,6 +19,7 @@ LXC_IP="$4"
 GEO_DBNAME="geonode_dev"
 GEO_DBUSER="geonode_dev"
 GEO_DBPWD="geonode_dev"
+GEO_COMMIT_HASH="1c65c9b"
```

Comment:
That is a commit hash it is almost self-evident, that is the hash that we use less.
Please substitute `COMMIT` with `STABLE` to have a more self-describing variable name.